src/components/KonnectorErrors.spec.jsx

```diff
@@ -29,6 +29,7 @@ describe('KonnectorErrors', () => {
         installedKonnectors={[{ slug: 'test', name: 'Test Konnector' }]}
         history={mockHistory}
         client={mockClient}
+        breakpoints={{ isMobile: false }}
```

Comment:
Should we add a test / snapshot for the mobile version?

Reply:
I don't think it's worth it, the only thing that changes based on this prop is the size of the button.